lib/src/kvs/tx.rs

```diff
@@ -1040,15 +1047,25 @@ impl Transaction {
 		}
 	}
 
-	// Public for tests, but we might not want to expose this
-	pub fn clock(&self) -> Timestamp {
+	/// Clock retrieves the current timestamp which is fallible
```

Comment:
Why is it described as "fallible" although the return type is not a `Result`?

Reply:
Removed fallible, and corrected to "not monotonic"